project01/paulo/apple.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <cinder/gl/gl.h>
+
+class GameCanvas;
+
+class Apple
+{
+public:
+  explicit Apple(GameCanvas&);
+  ~Apple();
+
+  const ci::ivec2& Location() const { return mLocation; }
+  void Draw();
+private:
+  GameCanvas& mCanvas;
```

Comment:
Is it good for Apple to know about / use canvas? This creates coupling - maybe it is better if someone else glue everything together